src/openrct2-ui/windows/NewsOptions.cpp

```diff
@@ -8,286 +8,285 @@
  *****************************************************************************/
 
 #include <cstddef>
-#include <iterator>
 #include <openrct2-ui/interface/Widget.h>
 #include <openrct2-ui/windows/Window.h>
 #include <openrct2/config/Config.h>
 #include <openrct2/drawing/Drawing.h>
-#include <openrct2/localisation/Localisation.h>
 #include <openrct2/sprites.h>
 
 static constexpr const rct_string_id WINDOW_TITLE = STR_NOTIFICATION_SETTINGS;
 static constexpr const int32_t WH = 300;
 static constexpr const int32_t WW = 400;
 
-// clang-format off
```

Comment:
Can you add this comment back along with the corresponding "on" some lines below and revert the indentation changes on the lines it was turned off? They were here on purpose because the big arrays are easier to read and review when each column is aligned.

Reply:
Sure, that makes sense. And thanks! I'm used to waiting for PRs so no worries about that. :^)